pkg/transforms/transformer.go

```diff
@@ -58,6 +58,7 @@ type Node struct {
 	ResourceString string                 `json:"resourceString"`
 	Properties     map[string]interface{} `json:"properties"`
 	Metadata       map[string]string
+	Time           int64
```

Comment:
Let’s qualify Time to avoid confusion. Maybe `changedTime` or `observedTime` ?

Reply:
+1